launcher.py

```diff
@@ -137,14 +147,23 @@ def validate_inputs(self, choices, module):
 
     # Execute commands as subprocesses
     def subprocess(self):
-        print("Launching... ", end="")
+        print(f"Launching... Run commands: {self.commands}")
         for command in self.commands:
+            launch_command = command[:-1]
+            std_flag = command[-1]
```

Comment:
I don't think adding a boolean onto the command is a good way to do this. Perhaps making a class for this, having both the command and the stdout flag, would be cleaner?